### ISSUE TITLE
Incompatibility with binary files in FileField/ImageField

### ISSUE THREAD

**Issue by Aksem** [Raekkeri/django-requestlogs#18]
The default implementation of request serializer that uses JSONEncoder is incompatible with FileField/ImageField that are supported by default in rest_framework(500 UnicodeDecodeError is raised).

In my project I've made a workaround by replacing JsonDumpField as follows:
```
class JsonDumpField(serializers.Field):
    def to_representation(self, value: QueryDict):
        """
        JSONEncoder doesn't support binary files such as images etc.
        Replace them with string 'File: <filename'
        """
        for val in value.items():
          if isinstance(val[1], InMemoryUploadedFile):
            value.update({ val[0]: f"File: {val[1].name}" })

        return json.dumps(value, cls=JSONEncoder)
```

The problem can be retrieved with the core functionality of `rest_framework`, that's why I think it should be solved in the package, not individually in a project.

**Comment by Raekkeri** [comment]
I've encountered this also, and agree that this could be fixed in this package. In the meanwhile, one quick'n'dirty way to fix this is to add the file field name into the `SECRETS` list in settings.

**Comment by Raekkeri** [comment]
So thanks for your input, I'm attempting to fix this with a solution almost identical to yours: https://github.com/Raekkeri/django-requestlogs/pull/19

**Comment by Raekkeri** [comment]
This is fixed in 0.2.3.
